lightning/src/ln/onion_utils.rs

```diff
@@ -328,8 +328,9 @@ pub(crate) fn set_max_path_length(
 	.serialized_length()
 	.saturating_add(PAYLOAD_HMAC_LEN);
 
+	const OVERPAY_ESTIMATE_MULTIPLER: u64 = 3;
```

Comment:
Just checking if the change from 4 to 3 was intentional.

Reply:
Yeah, should've called this out but 3 felt cleaner somehow? As noted here https://github.com/lightningdevkit/rust-lightning/pull/3026#discussion_r1600627013, these numbers may want tuning.